datadog/append.go

```diff
@@ -33,14 +33,6 @@ func appendMetric(b []byte, m Metric) []byte {
 
 func appendTags(b []byte, tags []stats.Tag) []byte {
 	for i, t := range tags {
-		if t.Name == "http_req_path" {
```

Comment:
@achille-roussel this is the filter I wasn't sure we should remove. If you know it's not being used anymore we can clean it up. But I don't want to take this out and have a flood of requests flow through that were being held back.